### PR TITLE
STP-84: text input very basic component.

### PR DESCRIPTION
**Pull request by n3op2**
# What

The fist PR from the saga of new components, please refer for more details in the story -> https://digicatapult.atlassian.net/jira/software/projects/STP/boards/186?selectedIssue=STP-84

two new components have been added:
- Input
- FormGroup - form group is basically a wrapper so we can use form.data

## Screenhots
Screenshot 2023-10-19 at 10 55 50
Screenshot 2023-10-19 at 10 55 46
Screenshot 2023-10-19 at 10 55 43

### CODE REVIEW

**Comment by jonmattgray** [comment]
missing the export of the component in `src/index.ts`

File: src/FormGroup/FormGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export default {
+  title: 'Components/FormGroup',
+  component: FormGroup,
+  argTypes: {
+    onSubmit: {},
+  },
+}
+
+const DefaultStoryTemplate = (args: FormGroupProps) => (
+  <FormGroup onSubmit={args.onSubmit}>
```
Comment (jonmattgray) [inline src/FormGroup/FormGroup.stories.tsx:15]: could use the `action` function from Storybook https://github.com/digicatapult/ui-component-library/blob/bd84e0a6eaa896b86944d509abd54dd33ff9c522/src/Search/Search.stories.tsx#L31

Comment (jonmattgray) [reply src/FormGroup/FormGroup.stories.tsx:15]: then can show in the storybook actions tab when you submit e.g. (from Search component) 
image

Comment (jonmattgray) [reply src/FormGroup/FormGroup.stories.tsx:15]: instead of a window alert

Comment (n3op2) [reply src/FormGroup/FormGroup.stories.tsx:15]: thanks!

**Pull request merged**